pcf8591/pcf8591.go

```diff
@@ -0,0 +1,83 @@
+// Package pcf8591 implements a driver for the PCF8591 Analog to Digital/Digital to Analog Converter.
+//
+// Datasheet: https://www.nxp.com/docs/en/data-sheet/PCF8591.pdf
+package pcf8591 // import "tinygo.org/x/drivers/pcf8591"
+
+import (
+	"machine"
+
+	"errors"
+
+	"tinygo.org/x/drivers"
+)
+
+// Device wraps PCF8591 ADC functions.
+type Device struct {
+	bus     drivers.I2C
+	Address uint16
+	CH0     ADCPin
+	CH1     ADCPin
+	CH2     ADCPin
+	CH3     ADCPin
+}
+
+// ADCPin is the implementation of the ADConverter interface.
+type ADCPin struct {
+	machine.Pin
+	d *Device
+}
+
+// New returns a new MCP3008 driver. Pass in a fully configured SPI bus.
```

Comment:
Please update it to the PCF8591 version

Reply:
Thanks, that was copy/paste error on my part. :smile_cat: